app/views/index.scala.html

```diff
@@ -49,6 +62,9 @@
         </div>
     </div>
 
+
```

Comment:
Looks like there's a bunch of random newlines added throughout the file. Not a huge deal, but definitely try not to add changes that aren't necessary or related to what you're changing in the future. You should always check what you changed with `git diff <file-path>` and undo anything extra before committing!